ext/date/php_date.stub.php

```diff
@@ -120,14 +120,14 @@ function checkdate(int $month, int $day, int $year): bool {}
 
 /**
  * @refcount 1
- * @deprecated
  */
+#[\Deprecated]
 function strftime(string $format, ?int $timestamp = null): string|false {}
 
 /**
  * @refcount 1
- * @deprecated
  */
+#[\Deprecated]
 function gmstrftime(string $format, ?int $timestamp = null): string|false {}
```

Comment:
Since 8.1, cf. https://wiki.php.net/rfc/deprecations_php_8_1#strftime_and_gmstrftime

Maybe mention the alternatives in the message?

Reply:
I can help document at least my deprecations in a followup